lib/index.js

```diff
@@ -31,6 +45,57 @@ internals.implementation = function (server, options) {
                 return reply(Boom.unauthorized(null, 'Access Token'));
             }
     
+            var parts = parseCookies(authorization);
```

Comment:
Not a contributor here, I just implemented a custom jwt auth recently and have some input:

1. You can also use `request.state[nameOfCookie]` instead of parsing the raw header. Would be good to take of advantage of hapi's cookie handling here, you don't have to write such low-level code again. This gives you either a string with the cookie's value, *or* an array of strings if multiple cookies by the same name were sent by the client. I ran into a bug where this happened.

2. I'd make the name of the cookie configurable and not require it to be `access_token`, even though that's probably a good default.